### ISSUE TITLE
Install Firebase Analytics without any IDFA collection capability.

### ISSUE THREAD

**Issue by cjkolstad** [capacitor-community/firebase-analytics#92]
Firebase provides a way to use Firebase Analytics without any IDFA collection capability.  There is a subspec available that can be used in place of the normal 'Firebase/Analytics' podspec.

pod 'Firebase/AnalyticsWithoutAdIdSupport'

For more information, please see [this documentation](https://firebase.google.com/docs/analytics/configure-data-collection?authuser=0&platform=ios).

**Comment by brownoxford** [comment]
Thanks for the report @cjkolstad, can you tell me a little more about what problem you are trying to solve and how you envision this working, e.g., enabling/disabling IDFA support via a compile time flag in your project, etc.

**Comment by cjkolstad** [comment]
@brownoxford Yes, I was assuming some sort of compile-time flag would do the trick.  I assume this would need to be done in the iOS project since this is iOS-specific.  Using the subspec 'Firebase/AnalyticsWithoutAdIdSupport' will ensure that the AdSupport framework is not included in the app.

**Comment by Schwankenson** [comment]
Are there any news on that? I do not want to add the app tracking transparency confirmation to my app. So I need to disable IDFA usage in Firebase Analytics. But i don`t see any possibility to do disable IDFA usage.

**Comment by Pynka** [comment]
I was able to solve it with [Disable IDFA collection](https://github.com/robingenz/capacitor-firebase/tree/main/packages/analytics#disable-idfa-collection)
